dwa_local_planner/src/dwa_planner.cpp

```diff
@@ -329,7 +331,7 @@ namespace dwa_local_planner {
       path_align_costs_.setScale(1.0);
       // disable goal cost during turning because turning won't move the robot closer to the goal
       goal_costs_.setScale(0.0);
-      // disable alignment cost during turning because turning will inadvertedly move the nose off the path
+      // disable alignment cost during turning because turning will inadvertently move the nose off the path
```

Comment:
`inadvertently` -> `inevitably`

Reply:
google says it's the same

Reply:
They're not the same thing...

inadvertently - without intention; **accidentally**
inevitably - as is certain to happen; **unavoidably**